scripts/funcotator/data_sources/createLiftoverForHg38ToB37.sh

```diff
@@ -0,0 +1,567 @@
+	echo 'fdfd811849cc2fadebc929bb925902e5	3						----'
+	echo '094d037050cad692b57ea12c4fef790f	----				chr6_qbl_hap6'
+	echo '18c17e1641ef04873b15f40f6c8659a4	----				chr6_cox_hap2'
+	echo '1e86411d73e6f00a10590f976be01623	----				chrY'
+	echo '2a3c677c426a10e137883ae1ffb8da3f	----				chr6_dbb_hap3'
+	echo '3b6d666200e72bcc036bf88a4d7e0749	----				chr6_ssto_hap7'
+	echo '641e4338fa8d52a5b781bd2a2c08d3c3	----				chr3'
+	echo '9d51d4152174461cd6715c7ddc588dc8	----				chr6_mann_hap4'
+	echo 'd2ed829b8a1628d16cbeee88e88e39eb	----				chrM'
+	echo 'd89517b400226d3b56e753972a7cad67	----				chr17_ctg5_hap1'
+	echo 'efed415dd8742349cb7aaca054675b9a	----				chr6_mcf_hap5'
+	echo 'fa24f81b680df26bcfb6d69b784fbe36	----				chr4_ctg9_hap1'
+	echo 'fe71bc63420d666884f37a3ad79f3317	----				chr6_apd_hap1'
+}
+
+function printDiffTable() {
+
+  echo 'The following table summarizes the differences between'
+  echo "The Broad Institute's B37 reference and UCSC's HG19 reference:"
+  echo ''
+  printSequenceDictionaryComparison 
+  echo ''
+  echo 'The table is grouped by the MD5SUM of the contents of each'
+  echo 'contig.'
+  echo 'Entries that are blank indicate a contig that does not have'
+  echo 'a matching entry in the other reference (e.g. B37 NC_007605,'
+  echo 'the Epstein-Barr virus).'
+  echo 'Some major contigs contain differences (e.g. chr3).  This is'
+  echo 'generally because of masked IUPAC bases (N, rather than '
+  echo 'A, T, G, or C).'
+  echo 'However, care should be taken to ensure that the resulting'
+  echo 'liftover files are used judiciously'
+  echo ''
+}
+
+################################################################################
+
+# Do all the interactive stuff if the script is called by a user.
+# The bash equivalent to `if __name__ == '__main__':
+if ${ISCALLEDBYUSER} ; then
+  
+  #========================================
+
+  trap at_exit EXIT 
+  
+  #========================================
+
+  #Check given arguments:
+  if [[ $# -gt $MAXARGS ]] ; then
+    usage
+    exit 1
+  elif [[ $# -lt $MINARGS ]] ; then
+    usage
+    exit 2
+  fi
+
+  # Make sure we have all the required commands:
+  checkPrerequisites $PREREQUISITES 
+  r=$?
+  [[ $r -ne 0 ]] && exit 4
+
+  #----------------------------------
+  #Read args:
+
+  while [ $# -gt 0 ] ; do
+
+    case "$1" in
+      -h|--h|--help|-help|help)
+        usage;
+        exit 0;
+        ;;
+      --showDiffs)
+        printDiffTable
+        exit 0
+        ;;
+      *)
+        error "${SCRIPTNAME}: invalid option: $1" 
+        simpleUsage
+        echo "Try \`$SCRIPTNAME --help' for more information."
+        exit 3;
+        ;; 
+    esac
+
+    #Get next argument in $1:
+    shift
+  done
+
+  #----------------------------------
+  # Do real work here.
+
+  error "Retrieving baseline hg38->hg19 chain file..."
+  hg19LiftoverFile=$( makeTemp )
+  curl "${HG38_HG19_CHAIN_FILE_URL}" 2>/dev/null | gunzip > ${hg19LiftoverFile}
```

Comment:
For code clarity it may be helpful to change `hg19LiftoverFile` to `hg38LiftoverFile` here and below.